logger.go

```diff
@@ -173,4 +173,8 @@ type LoggerOptions struct {
 
 	// The time format to use instead of the default
 	TimeFormat string
+
+	// Color the output. Colored logs are only avaiable for io.Writers that
+	// are concretely instances of *os.File.
```

Comment:
Ditto to my above comment on platform support, if we can make this restriction only true on Windows we probably should.

Reply:
Addressed this comment in the below commits. :)